packages/frontend/src/components/profile/export_private_key/ExportKeyModal.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import CheckMarkNoBorderIcon from '../../../images/icon-check-no-border.svg';
+import { selectAccountId } from '../../../redux/slices/account';
+import isMobile from '../../../utils/isMobile';
+import FormButton from '../../common/FormButton';
+import Modal from '../../common/modal/Modal';
+
+const Container = styled.div`
+    &&&&& {
+        padding: 15px 0 10px 0;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        text-align: center;
+
+        h3 {
+            margin: 15px 0;
+            font-size: 18px;
+            font-weight: 700;
+        }
+
+        p {
+            line-height: 1.5;
+            font-size: 14px;
+        }
+
+        label {
+            text-align: left;
+            display: flex;
+            background-color: #F5FAFF;
+            margin: 25px -25px 0 -25px;
+            padding: 15px 25px;
+            line-height: 1.5;
+
+            > div {
+                > div {
+                    border-color: #0081F1;
+                }
+            }
+
+            > span {
+                margin-left: 10px;
+                word-break: break-word;
+                color: #006ADC;
+            }
+
+            b {
+                color: #272729;
+            }
+        }
+
+        > button {
+            margin-top: 32px;
+            width: 100%;
+        }
+
+        .link {
+          margin-top: 20px;
+        }
+
+        .text-select-display {
+          display: flex;
+          flex-direction: row;
+          align-items: flex-start;
+          padding: 12px;
+          gap: 8px;
+
+          height: 96px;
+          width: 99%;
```

Comment:
Not sure why the gap, height, width is necessary? Can we remove that and just make padding 20px?